packages/katello/nodejs-compute-scroll-into-view/nodejs-compute-scroll-into-view.spec

```
@@ -0,0 +1,50 @@
+%{?scl:%scl_package nodejs-%{npm_name}}
+%{!?scl:%global pkg_name %{name}}
+
+%global npm_name compute-scroll-into-view
+
+Name: %{?scl_prefix}nodejs-compute-scroll-into-view
+Version: 1.0.14
+Release: 1%{?dist}
+Summary: The engine that powers scroll-into-view-if-needed
+License: MIT
+Group: Development/Libraries
+URL: https://scroll-into-view-if-needed.netlify.com
+Source0: https://registry.npmjs.org/%{npm_name}/-/%{npm_name}-%{version}.tgz
+%if 0%{?scl:1}
+BuildRequires: %{?scl_prefix_nodejs}npm
+%else
+BuildRequires: nodejs-packaging
```

This will likely be needed for el8 to work: (also in the other package)
```suggestion
BuildRequires: nodejs-packaging
BuildRequires: npm
```